rpc/openapi/openapi.yaml

```diff
@@ -54,6 +54,17 @@ info:
     However, the support for unsuffixed methods will be dropped in future releases and all users are
     encouraged to migrate as soon as possible.
 
+    ## Encoding
```

Comment:
Is it not base64?

Reply:
What are you referring to? If you're speaking about byte arrays, then yes.

Reply:
I know we have to encode txs in base64 format, this is the amino JSON format?